app/components/access_panels/location_item_component.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= tag.tr class: classes, data: { barcode: item.barcode } do %>
+  <td class="callnumber">
```

Comment:
This PR adds css classes to the `td` elements on the index page, but they appear to be unused on the index pages. The styles seem to come from the `availability-item` class applied exclusively by the show page.